.pylintrc

```diff
@@ -0,0 +1,2 @@
+[Format]
+max-line-length=80
```

Comment:
> I think the 80 (or 90) character limitation goes beyond screen space capacity. It enhances the readability and ensures that the lines won't wrap on any screen (maybe ones that are not larger-than-standard).

It depends a lot on your angle. To me (and many other people), limiting the line size to 80 chars on a screen that is capable of much more than that does **not** enhance readability, quite the contrary. Because of this, I believe that we shouldn't enforce such a limit.

> Many style rules are "pseudo-arbitrary" (vestiges of some old tech, or the inventor, or whatever), but I think the standardization that adhering to these rules provides is the key.

I don't. Style rules are arbitrary by definition, yes, but there are different levels of arbitrary out there. Check http://pylint-messages.wikidot.com/all-codes. Some of the things there are really arbitrary, and being forced to respect them might require completely reformatting my source code. Which seems rather unnecessary.

> I don't see why it would make contributors upset.

Because it is not OK for us to proclaim that we define The One Set of True Style Rules that anybody must adhere to. This is an open source project and it has various contributors with various backgrounds. I believe ths more relaxed style of contribution has worked quite well so far, so I really don't see a reason to start enforcing hard style rules out of nowhere. Getting warning for them is fine. Gating the acceptance of a PR because of them is not.